src/hooks/useOktaAxios.js

```diff
@@ -8,7 +8,7 @@ export default function useOktaAxios() {
   return axios.create({
     baseURL: process.env.REACT_APP_BACKENDURL || '',
     headers: {
-      authorization: isAuthenticated ? `${idToken.value}` : '',
+      authorization: isAuthenticated ? `Bearer ${idToken.value}` : '',
```

Comment:
I didn't see this in your explanation. Could I ask what this is for?

Reply:
This piece of code was to be removed, was going to be in a different PR by Ahmed but needed it to test code. Removed in recent push

Reply:
Hmmm I'm confused by that part though, you should keep Bearer in there, because if you remove it and that gets pushed, that will actually break this

Reply:
We added that a while back to  get auth working Ethan, alongside Ryan. It gets matched out in the authRequired function on the backend, without it the code breaks. This is in here on his end just leftover from the pair programming we were doing, but that's already merged in. So technically, do not remove that, make sure that still appears in this specific code on this branch.